views/012_changelog.sql

```diff
@@ -0,0 +1,75 @@
+-- Trigger def
+CREATE OR REPLACE FUNCTION change_trigger() 
+RETURNS trigger AS $$
+BEGIN
+  IF TG_TABLE_NAME = 'component_relationships' OR TG_TABLE_NAME = 'config_component_relationships' THEN
```

Comment:
Primary key would be: https://github.com/flanksource/duty/blob/main/schema/components.hcl#L105